### PR TITLE
Rohit2b/helm commands

### PR DESCRIPTION
**Pull request by rohit2b**
Document helper Helm commands

### CODE REVIEW

File: use-helm/README.rst [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+==================================
+Manage Helm packages in repository
+==================================
+
+A Helm chart repository is a location where packaged charts are stored and shared. The Confluent Operator Helm chart is 
+stored at https://confluent.jfrog.io/confluent/helm-early-access-operator-2
+
+Add the Confluent chart repository to your Helm repository:
+
+::
```
Comment (arodoni) [inline use-helm/README.rst:21]:
```suggestion
::

```

File: use-helm/README.rst [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+   helm repo add confluentinc_earlyaccess \   
+       https://confluent.jfrog.io/confluent/helm-early-access-operator-2 \
+       --username $USER \
+       --password $APIKEY
+  
+View all helm charts in your Helm repository:
+
+::
```
Comment (arodoni) [inline use-helm/README.rst:29]:
```suggestion
::

```

File: use-helm/README.rst [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+   helm repo list
+
+Update all Helm charts in repository to get latest versions:
+
+::
+   helm repo update
+
+Remove a specific helm chart from your Helm repository
+
+::
```
Comment (arodoni) [inline use-helm/README.rst:39]:
```suggestion
::

```

File: use-helm/README.rst [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+Deploy and Manage Helm charts
+=============================
+
+You deploy Helm charts to a release on your Kubernetes.
+
+helm install <release> <chart>
+
+There are two ways to deploy Confluent Operator chart to your Kubernetes:
+
+::
```
Comment (arodoni) [inline use-helm/README.rst:52]:
```suggestion
::

```

File: use-helm/README.rst [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+   # Install for the first time to release name "operator"
+   helm install operator confluentinc_earlyaccess/confluent-operator
+
+   # Upgrade to latest version, or install if the release does not exist
+   helm upgrade --install operator confluentinc_earlyaccess/confluent-operator
+
+Install a specific version of the Helm chart:
+
+::
```
Comment (arodoni) [inline use-helm/README.rst:61]:
```suggestion
::

```

File: use-helm/README.rst [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+   helm install operator confluentinc_earlyaccess/confluent-operator --version 0.72.0
+
+View all installed Helm releases on your Kubernetes:
+
+::
```
Comment (arodoni) [inline use-helm/README.rst:66]:
```suggestion
::

```

File: use-helm/README.rst [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+   helm list
+
+Delete the Helm release from your Kubernetes:
+
+::
```
Comment (arodoni) [inline use-helm/README.rst:71]:
```suggestion
::

```

File: use-helm/README.rst [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+
+Add the Confluent chart repository to your Helm repository:
+
+::
+   helm repo add confluentinc_earlyaccess \   
+       https://confluent.jfrog.io/confluent/helm-early-access-operator-2 \
+       --username $USER \
+       --password $APIKEY
+  
+View all helm charts in your Helm repository:
```
Comment (arodoni) [inline use-helm/README.rst:27]:
```suggestion
View all Helm charts in your Helm repository:
```

File: use-helm/README.rst [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+
+::
+   helm repo list
+
+Update all Helm charts in repository to get latest versions:
+
+::
+   helm repo update
+
+Remove a specific helm chart from your Helm repository
```
Comment (arodoni) [inline use-helm/README.rst:37]:
```suggestion
Remove a specific Helm chart from your Helm repository
```

File: use-helm/README.rst [hunk 10]
```diff
@@ -0,0 +1,74 @@
...
+::
+   helm repo remove confluentinc_earlyaccess
+
+=============================
+Deploy and Manage Helm charts
+=============================
+
+You deploy Helm charts to a release on your Kubernetes.
+
+helm install <release> <chart>
```
Comment (arodoni) [inline use-helm/README.rst:48]:
```suggestion
::

   helm install <release> <chart>
   
```

**Review by arodoni — CHANGES_REQUESTED** [review]

**Comment by rohit2b** [comment]
Thanks @arodoni ! Incorporated your review suggestions. Could you re-review?

**Review by arodoni — APPROVED** [review]
LGTM!

**Pull request merged**